app/controllers/assessments_controller.rb

```diff
@@ -594,7 +594,7 @@ def parseScore(feedback)
 
     score_hash = JSON.parse(feedback)
     score_hash = score_hash["scores"]
-    if @jsonFeedback.key?("_scores_order") == false
+    if @jsonFeedback && @jsonFeedback.key?("_scores_order") == false
```

Comment:
NIT: You could perhaps use ruby's safe navigator:
https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

Reply:
Note: if `@jsonFeedback` is `nil` then `@jsonFeedback&.key?("_scores_order")` would be `nil`, which is not the same as `false`

Reply:
Currently it also relies on the behavior of `nil` ANDed, which is also not very ideal.